examples/overhead_bench.py

```diff
@@ -0,0 +1,90 @@
+import argparse
+import logging
+import time
+
+import torch
+from llama import Llama
+
+from flex_model.core import FlexModel, HookFunction
+from flex_model.utils import setup_logger
+
+
+logger = logging.getLogger(__name__)
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--log-level", type=str, default="info")
+    parser.add_argument(
+        "--checkpoint-dir", type=str, default="/model-weights/Llama-2-13b"
+    )
+    parser.add_argument(
+        "--tokenizer-dir",
```

Comment:
Pedantic but maybe change this to `--tokenizer-path` if the argument expects the path to the tokenizer model, rather than its directory